client/src/App.js

```diff
@@ -1,55 +1,65 @@
-import React from 'react';
-import './components/TabNav';
-import TabNav from './components/TabNav';
-import PitContent from './components/PitContent';
-import ScoutContent from './components/ScoutContent';
-import AnalystContent from './components/AnalystContent';
-import 'bootstrap/dist/css/bootstrap.min.css';
+import React from "react";
+import "./components/TabNav";
+import TabNav from "./components/TabNav";
+import PitContent from "./components/PitContent";
+import ScoutContent from "./components/ScoutContent";
+import AnalystContent from "./components/AnalystContent";
+import "bootstrap/dist/css/bootstrap.min.css";
 //import 'bootstrap/dist/css/bootstrap.css';
-import './App.css';
+import "./App.css";
 
-function RenderTabContent({selectedTab}) {
-  if (selectedTab === 'pit') {
+window.onbeforeunload = function(e) {
+  return "";
+};
+
+function RenderTabContent({ selectedTab }) {
+  if (selectedTab === "pit") {
     return <PitContent />;
-  } else if (selectedTab === 'scout') {
+  } else if (selectedTab === "scout") {
     return <ScoutContent />;
   } else {
     return <AnalystContent />;
   }
 }
 
 class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      apiResponse:"",
-      selectedTab: ''
-    };
-  }
+  state = {
```

Comment:
Why was this taken out of a constructor?

Reply:
Let's test this to make sure the state is initialized on `this` before we set new values to `apiResponse` and `selectedTab`. This should still work but might affect React's _reactivity_